### PR TITLE
Updates to this repository.

### PR DESCRIPTION
**Pull request by xvzcf**
Updated this repository as per discussions with @dstebila and @baentsch. 

I've also added CMake and ninja to the CI environments, and once the environments are in use, I'll update the CI scripts in my CMake PR.

### CODE REVIEW

File: push-img.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+BASENAME=$1
+LOCALIMAGENAME=oqs-"${BASENAME}"
+
+cd "${BASENAME}"
+
+docker build -t "${LOCALIMAGENAME}"
+
+printf "Pushing image to openqsafe."
+docker tag "${LOCALIMAGENAME}" openquantumsafe/ci-"${BASENAME}"
+docker push openquantumsafe/ci-"${BASENAME}"
```
Comment (dstebila) [inline push-img.sh]: To be clear on the repository and tag naming, we would then have a repository `openquantumsafe/ci-centos-8` with a `oqs-centos-8` tag in it?

File: README.md [hunk 2]
```diff
@@ -1,103 +1,27 @@
...
 - Christian Paquin (Microsoft Research)
 - Douglas Stebila (University of Waterloo)
-- Ben Davies (University of Waterloo)
+- Goutam Tamvada (University of Waterloo)
 
 ### Support
 
 Financial support for the development of Open Quantum Safe has been provided by Amazon Web Services and the Tutte Institute for Mathematics and Computing.
-
 We'd like to make a special acknowledgement to the companies who have dedicated programmer time to contribute source code to OQS, including Amazon Web Services, evolutionQ, and Microsoft Research.
```
Comment (dstebila) [inline README.md]: Add Cisco Systems and IBM Research (in alphabetical order).

File: push-img.sh (hunk 1 above)
Comment (xvzcf) [reply push-img.sh]: No, the names I used were just placeholders.  I'll think about it some more and revise it.

**Comment by dstebila** [comment]
Thinking about Dockerhub repository names some more, I think we should go with `openquantumsafe/test-ubuntu-bionic-amd64`, `openquantum-safe/dev-ubuntu-bionic-amd64`, `openquantum-safe/run-ubuntu-bionic-amd64` etc., with version numbers as tags.  This is because if you just do a `docker run organization/repository` command, it will try to use the `:latest` tag automatically.

File: centos-7/build-openssl111d.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+cd /root
+git clone https://github.com/openssl/openssl.git
```
Comment (dstebila) [inline centos-7/build-openssl111d.sh]: Can save bandwidth by doing `git clone --single-branch`. Or could just download the OpenSSL release .tar.gz from their website.

Comment (xvzcf) [reply centos-7/build-openssl111d.sh]: It was downloading the `.tar.gz` before, but I was running into build errors involving `zlib`, even with the `no-comp` option passed to `./config`. For now I just decided to switch to the source, but I'll add a TODO to look into this apparent discrepancy.

**Pull request merged**